lib/methods.js

```diff
@@ -125,26 +125,21 @@ swap.inline = (options, selectors = []) => {
 }
 
 
-const createAndTriggerButtonForm = (btn, action) => {
-  const form = document.createElement('form');
-  form.method = btn.getAttribute('formmethod');
-  form.action = action;
-  document.body.appendChild(form);
-  form.submit();
-}
-
-
 swap.submit = function(e, selectors) {
-  const target = e.target; // form, submit button, or swap button form
+  const target = e.target; // form, submit button, or button form
   const formaction = target.getAttribute('formaction');
   const action = formaction || target.action || target.href;
 
   if (!shouldSwap(getUrl(action))) {
     // when action is on different hostname than location.hostname
     if (formaction) {
-      // this is a buttonForm use case
+      // ceate and trigger buttonForm on different hostname
```

Comment:
This comment is a little confusing, especially in light of the comment above.

How about:
```
// create and trigger buttonForm when on a different hostname
```

Reply:
I agree. How about `create and trigger buttonForm for remote hostname`?

Reply:
If you don't like `remote` we could do `external`

Reply:
👍 

external is probably more clear.

You should probably update the above comment in light of that as well then.

Reply:
`// when action points at an external hostname`
`// create and trigger buttonForm for external hostname`